### DISCUSSION TITLE
Repeater additions

### DISCUSSION THREAD

**Discussion by phpsa** [filamentphp/filament#2499] [Ideas]
Would be nice if any or all of the following are possible:

1 - By default all repeaters minimised - expand what you want 
2 - when minimized allow a field's state to be in the header of the repeater

**Comment by phpsa** [comment]
If a few votes come in - I may open a PR for this -- Or if @danharrin gives it a yes.. :-)

**Comment by Zagna** [comment]
I was thinking about 2 as well https://github.com/laravel-filament/filament/discussions/2429 
I put the changes I came up with [here](https://github.com/Zagna/filament/commits/repeater-header)
And my way feels wrong since it needs to come after `schema`, only works with TextInput and Select and overall jank.

**Comment by danharrin** [comment]
First one is a good idea, second one is probably tricky though...

**Reply by phpsa** [reply]
https://github.com/laravel-filament/filament/pull/2501 for the first one,
